PubHub/PubHub.Common/Services/UserService.cs

```diff
@@ -55,12 +57,12 @@ public async Task<string> AddUser(UserCreateModel userCreateModel)
                     throw new Exception($"Unable to retrieve information: {errorResponse!.Detail}");
                 }
 
-                return $"Successfully added the user: {userCreateModel.Name}";
+                return new ServiceResult<UserCreateModel>(response.StatusCode.ToString(), userCreateModel, $"Successfully added the user: {userCreateModel.Name}");
```

Comment:
No need to make it `ToString()` you can just store it as an int, which it is already (: